Workbooks/SapMonitor/SapNetWeaver/SapNetWeaver.workbook

```diff
@@ -2592,7 +2389,7 @@
                   "multiSelect": true,
                   "quote": "'",
                   "delimiter": ",",
-                  "query": "SapNetweaver_GetSystemInstanceList_CL\r\n| where SID_s == '{QueueStatisticsSID}' or '{QueueStatisticsSID}' == ''\r\n| extend instanceNr_s = tostring(toint(instanceNr_d))\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 2, instanceNr_s, strcat( \"0\", instanceNr_s))\r\n| distinct SID_s, hostname_s, instanceNr_s\r\n| project AppServer = strcat(hostname_s, \"_\", instanceNr_s)\r\n| distinct AppServer",
+                  "query": "SapNetweaver_GetSystemInstanceList_CL\r\n| where SID_s == '{QueueStatisticsSID}' or '{QueueStatisticsSID}' == ''\r\n| where serverTimestamp_t between (todatetime({QueueStatisticsTimeRange:start}) .. todatetime({QueueStatisticsTimeRange:end}))\r\n| extend instanceNr_s = tostring(toint(instanceNr_d))\r\n| extend instanceNr_s = iff(strlen(instanceNr_s) == 2, instanceNr_s, strcat( \"0\", instanceNr_s))\r\n| distinct SID_s, hostname_s, instanceNr_s\r\n| project AppServer = strcat(hostname_s, \"_\", instanceNr_s)\r\n| distinct AppServer",
```

Comment:
Kavitha, still confused as this is referencing serverTimestamp_t yet other queries are referencing timestamp_t

Reply:
Here you are using serverTimestamp_t when you query SapNetweaver_GetSystemInstanceList_CL, but below on line 2746 you are using timestamp_t when you query SapNetweaver_GetQueueStatistic_CL.  Why are they using datetime properties?

Reply:
Won't fix in this PR and will investigate more if this requires a fix.